static/panes/editor.js

```diff
@@ -706,6 +706,29 @@ Editor.prototype.initEditorActions = function () {
             this.tryPanesLinkLine(ed.getPosition().lineNumber, true);
         }, this)
     });
+
+    this.isCpp = this.editor.createContextKey('isCpp', true);
+    this.isCpp.set(this.currentLanguage.id === 'c++');
+
+    this.editor.addAction({
+        id: 'cpprefsearch',
+        label: 'Search on Cppreference',
+        keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.F8],
+        keybindingContext: null,
+        contextMenuGroupId: 'help',
+        contextMenuOrder: 1.5,
+        precondition: 'isCpp',
+        run: _.bind(this.searchOnCppreference, this)
+    });
+};
+
+Editor.prototype.searchOnCppreference = function (ed) {
+    var pos = ed.getPosition();
+    var word = ed.getModel().getWordAtPosition(pos);
+    if (!word || !word.word) return;
+
+    var url = "https://en.cppreference.com/mwiki/index.php?search=" + encodeURIComponent(word.word);
+    window.open(url, '_blank');
```

Comment:
We should add `noopener` here - I believe in js it just goes in the third parameter to window.open, but please double check me on that.

Reply:
Good one, will do

Reply:
Checked mozilla docs and its indeed the third and accepts 'noopener'